ggshield/sca/output/text_handler.py

```diff
@@ -180,17 +358,67 @@ def sca_incident_fix_version_line(vulnerability: SCAVulnerability) -> str:
     return "No fix is currently available.\n"
 
 
-def sca_incident_cve_ids(vulnerability: SCAVulnerability) -> str:
+def sca_incident_cve_ids(vulnerability: OutputIncidentData) -> str:
     if not vulnerability.cve_ids:
         cve_str = "-"
     else:
         cve_str = ", ".join(vulnerability.cve_ids)
     return f"CVE IDs: {cve_str}\n"
 
 
-def sca_incident_identifier(vulnerability: SCAVulnerability) -> str:
+def sca_incident_identifier(vulnerability: OutputIncidentData) -> str:
     identifier = vulnerability.identifier
     # TODO Remove clause when identifier field is in production on backend side (v2.36)
     if identifier is None:
         return ""
     return f"Identifier: {identifier}\n"
+
+
+def diff_scan_summary(result: SCAScanDiffOutput) -> str:
+    added_counter = Counter(
+        vuln.severity
+        for location in result.added_vulns
+        for pkg in location.package_vulns
+        for vuln in pkg.vulns
+    )
+    num_added = sum(added_counter.values())
+
+    removed_counter = Counter(
+        vuln.severity
+        for location in result.removed_vulns
+        for pkg in location.package_vulns
+        for vuln in pkg.vulns
+    )
+    num_removed = sum(removed_counter.values())
+
+    def label_incident(n: int) -> str:
```

Comment:
we don't really need this method do we?

Reply:
Indeed, it's just to avoid hard-coding several times the same strings.